py/test/selenium/webdriver/remote/custom_element_tests.py

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import pytest
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.remote.webelement import WebElement
+
+
+# Custom element class
+class MyCustomElement(WebElement):
+    def custom_method(self):
+        return "Custom element method"
+
+
+@pytest.fixture
+def driver():
+    options = webdriver.ChromeOptions()
```

Comment:
Any reason to avoid the existing `driver` fixture? https://github.com/SeleniumHQ/selenium/blob/f4518b89563fc83afdc16f49455731987dfbafce/py/conftest.py#L103-L183

Reply:
No specific reason, I will update this to utilize the existing driver fixture.